### DISCUSSION TITLE
Intent 9 analysis

### DISCUSSION THREAD

**Discussion by Kevsy** [camaraproject/EdgeCloud#180] [General]
### Developer intents: Provisioning intents

| Intent 9 | 5SED |5MEE |EXRA|EXRC| EXCA | EXSM|  GMEC |
| -------- |------| ----|----| ---|------|-----|-------|
| 9. "I can ask the operator to inform me if the optimal MEC for my application and a specific terminal changes, taking into account mobility events, connectivity, shortest network path, cost, network load, MEC platform load etc."  | No           |No    | ?          | ?     | ?            | ?     | Yes   |

| GMEC           | Details       |Comments|
| --------------|-------------| -----------|
| (to complete) | (to complete) | (to complete) |

(others add as appropriate) |

**Comment by maheshc01** [comment]
the version of the API spec contributed does not support user subscriptions and notifications but 5GFF has an updated API spec which supports this and the plan is to contribute it to CAMARA as a generic module which can be use across multiple APIs

**Comment by gunjald** [comment]
This intent i.e., intent 9 seems same as intent mention in the issue #112. Is that an error?
